ROMFS/px4fmu_common/init.d/airframes/4650_generic_quad_650

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# @name Generic Quadcopter (650mm)
+#
+# @type Quadrotor x
+# @class Copter
+#
+
+sh /etc/init.d/rc.mc_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	param set MC_ROLL_P 6.5
+	param set MC_ROLLRATE_P 0.05
+	param set MC_ROLLRATE_I 0.05
+	param set MC_ROLLRATE_D 0.001
+
+	param set MC_PITCH_P 6.5
+	param set MC_PITCHRATE_P 0.05
+	param set MC_PITCHRATE_I 0.05
```

Comment:
Please keep all I gains default (only where set higher than the default (0.2), you can keep it)